dali/image/tiff_libtiff_impl.h

```diff
@@ -0,0 +1,51 @@
+// Copyright (c) 2019, NVIDIA CORPORATION. All rights reserved.
```

Comment:
This file can be renamed to `tiff_libtiff.h`, cause this is by definition an implementation. No need to explicitly state it